letsdrill.rb

```diff
@@ -1,12 +1,23 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+   if integer < 60
+    "F"
+  elsif integer < 70
+    "D"
+  elsif integer < 80
+    "C"
+  elsif integer < 90
+    "B"
+  else
+    "A"
+  end
 
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  sorted = array.sort { | x, y | x.length <=> y.length }
+  sorted[0]
```

Comment:
This is pretty solid.  But take a look through the ruby Enumerable docs and see if you can find an even more apt choice of method.